### PR TITLE
docs: fix typo. errors for "hasura seed" => "hasura seeds"

### PR DESCRIPTION
**Pull request by codeliezel**
### Description
This PR fixes typo errors for Hasura seeds.

### Changelog

- [x] `CHANGELOG.md` is updated with user-facing content relevant to this PR. If no changelog is required, then add the `no-changelog-required` label.

### Affected components
- [x] Docs

### Related Issues
https://discord.com/channels/407792526867693568/428469959530643466/778553964089180200

### Affected pages
1. https://deploy-preview-6245--hasura-docs.netlify.app/graphql/core/hasura-cli/hasura_seed_apply.html
2. https://deploy-preview-6245--hasura-docs.netlify.app/graphql/core/hasura-cli/hasura_seed_create.html
3. https://deploy-preview-6245--hasura-docs.netlify.app/graphql/core/hasura-cli/hasura_seed.html

### CODE REVIEW

**Review by marionschleifer — CHANGES_REQUESTED** [review]
@codeliezel I just checked with @scriptonist and he said the actual command is `hasura seed`, so let's go with this one 🙂

File: docs/graphql/core/hasura-cli/hasura_seed_apply.rst [hunk 1]
```diff
@@ -29,7 +29,7 @@ Examples
     hasura seed apply
 
     # Apply only a particular file:
-    hasura seed apply --file seeds/1234_add_some_seed_data.sql
+    hasura seed apply --file seed/1234_add_some_seed_data.sql
```
Comment (marionschleifer) [inline docs/graphql/core/hasura-cli/hasura_seed_apply.rst:32]: @codeliezel have you actually tried this out? I don't know if the actual folder structure will change names.

**Review by marionschleifer — CHANGES_REQUESTED** [review]
@codeliezel please address my comment below. Also, here there's an image that shouldn't be here.

**Comment by codeliezel** [comment]
Moving to the mono-repo

**Pull request closed without merging**